lib/iris/tests/results/imagerepo.json

```diff
@@ -908,6 +908,10 @@
         "https://scitools.github.io/test-iris-imagehash/images/v4/bb433d4e94a4c6b9c15adaadc1fb6a469c8de43a3e07904e5f016b57984e1ea1.png",
         "https://scitools.github.io/test-iris-imagehash/images/v4/eea16affc05ab500956e974ac53f3d80925ac03f3f81c07e3fa12da1c27e3f80.png"
     ],
+    "iris.tests.test_quickplot.TestLabels.test_pcolormesh_str_symbol.0": [
+        "https://scitools.github.io/test-iris-imagehash/images/v4/bb433d4e94a4c6b9c15adaadc1fb6a469c8de43a3e07904e5f016b57984e1ea1.png",
```

Comment:
I'm unclear why this image has also been included as an acceptable result for `test_pcolormesh_str_symbol`. From the colormap it appears to have been created with an older version of Matplotlib that predates your change, and also **does not** demonstrate what you are aiming to demonstrate with the image you've added in SciTools/test-iris-imagehash#36:

* [bb433d4e94a4c6b9c15adaadc1fb6a469c8de43a3e07904e5f016b57984e1ea1.png](https://github.com/SciTools/test-iris-imagehash/blob/gh-pages/images/ddc2bc722925639d835a5bb583df566239b1275c7ce00972fa80d6ea19727885.png)
* [eea16affc05ab500956e974ac53f3d80925ac03f3f80c07e3fa12da1c27f3f80.png](https://github.com/SciTools/test-iris-imagehash/blob/12e3ab75ea119ee7fcc5c14ad051c4bb3768136e/images/v4/eea16affc05ab500956e974ac53f3d80925ac03f3f80c07e3fa12da1c27f3f80.png)

Perhaps this was a mishap while using `idiff.py`?

Reply:
Yep I think I made a mess there, I have removed the wrong file so hopefully good to go now.